packages/app-account/src/interbit/control/index.js

```diff
@@ -39,6 +39,19 @@ const reducer = (state = initialState, action) => {
   }
 }
 
+const getCovenantHash = (covenantAlias, manifest) => {
+  if (!manifest.covenants) {
+    return undefined
+  }
+
+  const covenant = manifest.covenants[covenantAlias]
+  if (typeof covenant === 'string') {
```

Comment:
some are string, some are not?

Reply:
If it is string (covenant hash) then it may be a system covenant but the expectation is the covenant is already loaded. If it is an object, then we need additional info to load it.

On reflection, these should always be objects with a covenantHash property. It is just the file path that becomes optional.